pkg/configprofiles/profiles.go

```diff
@@ -0,0 +1,177 @@
+		tasks: append(
+			multitenantClusterInitTasks,
+			makeTask("create an application tenant",
+				nil, /* nonTxnSQL */
+				/* txnSQL */ []string{
+					// Create the app tenant record.
+					"CREATE TENANT application",
+					// Run the service for the application tenant.
+					"ALTER TENANT application START SERVICE SHARED",
+				},
+			),
+			makeTask("activate application tenant",
+				/* nonTxnSQL */ []string{
+					// Make the app tenant receive SQL connections by default.
+					"SET CLUSTER SETTING server.controller.default_tenant = 'application'",
+				},
+				nil, /* txnSQL */
+			),
+		),
+	},
+}
+
+var multitenantClusterInitTasks = []autoconfigpb.Task{
+	makeTask("initial cluster config",
+		/* nonTxnSQL */ []string{
+			// Disable RU limits.
+			"SET CLUSTER SETTING kv.tenant_rate_limiter.burst_limit_seconds = 10000000",
+			"SET CLUSTER SETTING kv.tenant_rate_limiter.rate_limit = -1000",
+			"SET CLUSTER SETTING kv.tenant_rate_limiter.read_batch_cost = 0",
+			"SET CLUSTER SETTING kv.tenant_rate_limiter.read_cost_per_mebibyte = 0",
+			"SET CLUSTER SETTING kv.tenant_rate_limiter.write_cost_per_megabyte = 0",
+			"SET CLUSTER SETTING kv.tenant_rate_limiter.write_request_cost = 0",
+			// Disable trace redaction (this ought to be configurable per-tenant, but is not possible yet in v23.1).
+			"SET CLUSTER SETTING server.secondary_tenants.redact_trace.enabled = false",
+			// Enable zone config changes in secondary tenants  (this ought to be configurable per-tenant, but is not possible yet in v23.1).
+			"SET CLUSTER SETTING sql.zone_configs.allow_for_secondary_tenant.enabled = true",
+			// Enable multi-region abstractions in secondary tenants.
+			"SET CLUSTER SETTING sql.multi_region.allow_abstractions_for_secondary_tenants.enabled = true",
+			// Disable range coalescing (as long as the problems related
+			// to range coalescing have not been solved yet).
+			"SET CLUSTER SETTING spanconfig.storage_coalesce_adjacent.enabled = false",
+			"SET CLUSTER SETTING spanconfig.tenant_coalesce_adjacent.enabled = false",
+			// Make the operator double-check tenant deletions.
+			"SET CLUSTER SETTING sql.drop_tenant.enabled = false",
+		},
+		nil, /* txnSQL */
+	),
+	makeTask("create tenant template",
+		nil, /* nonTxnSQL */
+		[]string{
+			// Create a main secondary tenant template.
+			"CREATE TENANT template",
+			"ALTER TENANT template GRANT CAPABILITY can_admin_relocate_range, can_admin_unsplit, can_view_node_info, can_view_tsdb_metrics, exempt_from_rate_limiting",
+			// Disable span config limits and splitter.
+			// TODO(knz): Move this to in-tenant config task.
+			"ALTER TENANT template SET CLUSTER SETTING spanconfig.tenant_limit = 1000000",
+			// "ALTER TENANT template SET CLUSTER SETTING spanconfig.tenant_split.enabled = false",
+			// Disable RU accounting.
+			// TODO(knz): Move this to in-tenant config task.
+			"SELECT crdb_internal.update_tenant_resource_limits('template', 10000000000, 0, 10000000000, now(), 0)",
```

Comment:
I don't think we need this anymore since we won't have the RU limiter for shared-process tenants by default.

Reply:
Simplified.